packages/elements/src/overlay-menu/index.ts

```diff
@@ -910,6 +937,33 @@ export class OverlayMenu extends Overlay {
     // closes opened child menu if any
     // OpenedMenusManager.toggleNestedMenuFor(this);
     this.onMenuReHighlight();
+
+    const children = this.children;
```

Comment:
Move this code to a separate function.

As well, I am not sure if this should be handled inside our code, or the developer must provide these, but we just have to provide the documentation?

Reply:
Yeah I think we shouldn't handle this in our code. These attributes are just initialisations and it can be handled by developers themselves.

Reply:
Agreed. I will remove it.

Reply:
removed